src/Relation/Binary/Construct/Interior/Symmetric.agda

```diff
@@ -0,0 +1,95 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Symmetric interior of a binary relation
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Relation.Binary.Construct.Interior.Symmetric where
+
+open import Function.Base using (flip)
+open import Level
+open import Relation.Binary
+
+private
+  variable
+    a b c ℓ r s t : Level
+    A : Set a
+    R : Rel A r
+    S : Rel A s
+    T : Rel A t
+
+------------------------------------------------------------------------
+-- Definition
+
+record SymInterior (R : Rel A ℓ) (x y : A) : Set ℓ where
+  constructor _,_
+  field
+    lhs≤rhs : R x y
+    rhs≤lhs : R y x
+open SymInterior public
+
+------------------------------------------------------------------------
+-- Properties
+
+-- A reflexive transitive relation _≤_ gives rise to a poset in which the
+-- equivalence relation is SymInterior _≤_.
+record IsProset {a ℓ} {A : Set a} (_≤_ : Rel A ℓ) : Set (a ⊔ ℓ) where
+  field
+    refl : Reflexive _≤_
+    trans : Transitive _≤_
```

Comment:
Did you see my comments about the existence of `IsProset`/`Proset`?

Reply:
So, `IsProset` can't be moved to `Relation.Binary.Structures` because of the file-level module parameters (particularly `_≈_`). Perhaps a compromise is to expand it out in the definition of `Proset` and move that to `Relation.Binary.Bundles`. This way, the definition `poset` is retained.